components/input/input.css

```diff
@@ -0,0 +1,42 @@
+@import url('/components/css/common.css');
+form{
+    display: flex;
+    flex-direction: column;
+    padding:2rem;
```

Comment:
Spacing between property and value is not given. You can beautify file to make the code consistent.